_release_common.sh

```diff
@@ -1,5 +1,27 @@
 #!/bin/bash
 
+function commit_to_branch_and_send_pull_request {
```

Comment:
@kiwm, try to move these 2 functions to `_git.sh`. Also, check https://github.com/liferay/liferay-basic-training/blob/ebf76a7f7c2bb5fdc526a47f68d2cef777f13b89/dotfiles/os/restore.sh#L3 since it can help us during the import of other scripts.